examples/visual_recognition_v3.rb

```diff
@@ -22,7 +22,7 @@
 # trucks = File.open(Dir.getwd + "/resources/trucks.zip")
 # puts JSON.pretty_generate(visual_recognition.create_classifier(
 #   name: "Cars vs Trucks",
-#   classname_positive_examples: cars,
+#   positive_examples: { cars: cars,
```

Comment:
missing closing bracket

Reply:
oops, didn't catch that because it was in a comment :)